bitnami/mxnet/values.yaml

```diff
@@ -0,0 +1,216 @@
+## Global Docker image parameters
+## Please, note that this will override the image parameters, including dependencies, configured to use the global value
+## Current available global Docker image parameters: imageRegistry and imagePullSecrets
+##
+# global:
+#   imageRegistry: myRegistryName
+#   imagePullSecrets:
+#     - myRegistryKeySecretName
+
+## Bitnami MXNet image version
+## ref: https://hub.docker.com/r/bitnami/mxnet/tags/
+##
+image:
+  registry: docker.io
+  repository: bitnami/mxnet
+  tag: 1.4.0-r0
```

Comment:
This tag is a rolling tag, please use the one with the platform `1.4.0-debian-9-r0`

Reply:
Done